docs/source/table.rst

```diff
@@ -0,0 +1,135 @@
+==============
+Table
+==============
+
+.. currentmodule:: snowflake.snowpark
+
+.. rubric:: Classes
+
+.. autosummary::
+    :toctree: api/
+
+    Table
+
+.. rubric:: Methods
+
+.. autosummary::
+    :toctree: api/
+
+    Table.agg
```

Comment:
Table subclasses DataFrame so it inherits many methods. But actually it only have several methods specific to Table class, which are more important. Table class on the original API doc only has these methods, which is more clear I believe.

Reply:
I see, I copy/pasted the contents of the generate file, lemme remove the inherited methods then.